agent-inject/agent/annotations_test.go

```diff
@@ -424,24 +415,20 @@ func TestSecretMixedTemplatesAnnotations(t *testing.T) {
 	}{
 		{
 			map[string]string{
-				"vault.hashicorp.com/agent-inject-secret-foobar":        "test1",
-				"vault.hashicorp.com/agent-inject-template-foobar":      "",
```

Comment:
Please can we leave these test cases (here and below) in place and add new ones for the new behaviour?